saleor/graphql/payment/schema.py

```diff
@@ -51,7 +52,10 @@ class PaymentQueries(graphene.ObjectType):
         TransactionItem,
         description="Look up a transaction by ID." + ADDED_IN_36 + PREVIEW_FEATURE,
         id=graphene.Argument(
-            graphene.ID, description="ID of a transaction.", required=True
+            graphene.ID, description="ID of a transaction.", required=False
```

Comment:
You should mention here, that one of these is mandatory to fetch the transaction data